examples/python/Quantinuum_circuit_submissions.py

```diff
@@ -13,38 +16,31 @@
 #     * [Save Results](#save-results)
 #     * [Analyze Results](#analyze-results)
 #     * [Cancel Jobs](#cancel-jobs)
-# * [Additional Features](#additional-features)
-#     * [Extended Circuit Compilation](#circuit-compilation-extended)
-#     * [Batch Submission](#batch-submission)
-#     * [Parametrized Circuits](#parametrized-circuits)
-#     * [Conditional Gates](#conditional-gates)
-
-# ## What is TKET? <a class="anchor" id="step-by-step"></a>
+# ## What is TKET? <a class="anchor" id="tket"></a>
 
 # The TKET framework (pronounced "ticket") is a software platform for the development and execution of gate-level quantum computation, providing state-of-the-art performance in circuit compilation. It was created and is maintained by Quantinuum. The toolset is designed to extract the most out of the available NISQ devices of today and is platform-agnostic.
 
-# In python, the `pytket` packages is available for python 3.8+. The `pytket` and `pytket-quantinuum` packages are included as part of the installation instructions on the user portal.
+# In python, the `pytket` packages is available for python 3.9+. The `pytket` and `pytket-quantinuum` packages are included as part of the installation instructions on the user portal.
 
 # For more information on TKET, see the following links:
-# - [TKET user manual](https://tket.quantinuum.com/user-manual/manual_intro.html)
+# - [TKET user manual](https://cqcl.github.io/pytket/manual/manual_intro.html)
 # - [TKET overview and demo video](https://www.youtube.com/watch?v=yXKSpvgAtrk)
 # - [Quantum Compilation with TKET](https://calmaccq.github.io/tket_blog/tket_compilation.html)
```

Comment:
This blog page no longer exists. 

The blog is now at https://tket.quantinuum.com/blog/

I would suggest

For more information on TKET, see the following links:
 - [TKET user manual](https://tket.quantinuum.com/user-manual/manual_intro.html)
 - [TKET notebook examples](https://tket.quantinuum.com/examples/Getting_started.html)